assets/js/atomic/blocks/product-elements/image/block.js

```diff
@@ -69,37 +81,21 @@ const Block = ( {
 				}
 			) }
 		>
-			{ productLink ? (
-				<a href={ product.permalink } rel="nofollow">
-					{ !! showSaleBadge && (
-						<ProductSaleBadge
-							align={ saleBadgeAlign }
-							product={ product }
-						/>
-					) }
-					<Image
-						image={ image }
-						onLoad={ () => setImageLoaded( true ) }
-						loaded={ imageLoaded }
-						showFullSize={ imageSizing !== 'cropped' }
+			<ParentComponent { ...( showProductLink && anchorProps ) }>
+				{ !! showSaleBadge && (
```

Comment:
I see that it was here before but `showSaleBadge` has `PropTypes.bool` so I don't see a reason for `!!` maybe we can remove it?

Reply:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3756